src/InputTypeUtils.php

```diff
@@ -144,6 +145,9 @@ public static function getInputTypeArgs(array $args): array
             if ($parameter->hasDefaultValue()) {
                 $desc['defaultValue'] = $parameter->getDefaultValue();
             }
+            if ($parameter instanceof InputTypeParameter && $parameter->getDescription()) {
```

Comment:
Not sure if this is ok to use `instanceof` here. Maybe `getDescription` should be pulled up to `InputTypeParameterInterface`?

Reply:
@downace I think extending that interface makes sense.  There isn't much using it at the moment and it's tagged unstable anyway.  Why shouldn't `getName` also be included in the interface?

Reply:
> Why shouldn't getName also be included in the interface?

Sounds reasonable. Name is common for any Input parameter.